errtracker/errtracker.go

```diff
@@ -119,6 +130,8 @@ func Report(snap, errMsg, dupSig string, extra map[string]string) (string, error
 		"CoreSnapdBuildID":   coreBuildID,
 		"Date":               timeNow().Format(time.ANSIC),
 		"Snap":               snap,
+		"SnapConfineCurrent": snapConfineProfileDigest(""),
```

Comment:
Maybe `SnapConfineApparmorProfileCurrent` to avoid possible confusion with the hash of the actual snap-confine binary? Same below.